models/TaskAssignee.js

```diff
@@ -0,0 +1,20 @@
+const mongoose = require("mongoose");
+
+const TaskAssigneeSchema = new mongoose.Schema(
+	{
+		assigneeId: { type: mongoose.Schema.Types.ObjectId, ref: "User" },
+		assignedBy: { type: mongoose.Schema.Types.ObjectId, ref: "User" },
+		status: {
+			type: String,
+			default: "pending",
+			enum: ["done", "onit", "pending", "overdue"]
```

Comment:
can we have these options for tasks status?
`[
 "pending",
 "ongoing",
 "completed",
 "closed",
 "overdue"
]`

they look more modular to me

Reply:
Yes, they seem better